pkg/gitreceive/build.go

```diff
@@ -97,7 +100,7 @@ func build(conf *Config, s3Client *s3.S3, kubeClient *client.Client, builderKey,
 	bType := getBuildTypeForDir(tmpDir)
 	usingDockerfile := bType == buildTypeDockerfile
 
-	var procType pkg.ProcessType
+	procType := pkg.ProcessType{}
```

Comment:
Hm, never seen it do that. Anyway, they both set `procType` to a `ProcessType` filled in with all zero values. It would be good to know why the previous code didn't work but this does, if that's indeed the case.